cinder/scheduler/filter_scheduler.py

```diff
@@ -603,6 +603,9 @@ def _schedule(self,
                     else volume_utils.extract_host(backend.obj.backend_id)
                 )
                 if backend_id != resource_backend:
+                    LOG.warning("Backend '%s' removed because it doesn't "
+                                "match requested backend '%s'.",
```

Comment:
hm ... Can we put `resource_backend` in here i.e. "requested resource_backend"? Or maybe we can state "required backend" instead? Reading this log line, it wouldn't be clear to me that the driver requested this backend and not the user. So it would be easy to jump to "hey user, you need to request a different backend" as solution.

Reply:
updated